src/tests/unit/tests/injected/analyzers/rule-analyzer.test.ts

```diff
@@ -28,7 +28,7 @@ describe('RuleAnalyzer', () => {
     let scopingStoreMock: IMock<ScopingStore>;
     let scopingState: ScopingStoreData;
     let visualizationConfigurationFactoryMock: IMock<VisualizationConfigurationFactory>;
-    const mockAllInstances: DictionaryStringTo<any> = {
+    const allInstancesMocks: DictionaryStringTo<any> = {
         test: 'test-result-value',
```

Comment:
allInstancesMock(singular). this is just 1 mock not a collection of mocks

Reply:
make sense, updated!